scripts/update_op_coverage.py

```diff
@@ -91,7 +91,13 @@ def get_tested_ops_for_backends(build_dir):
   for t in tests:
     if not t.endswith('.mlir'):
       continue
-    backend, op = get_backend_op_pair(t)
+    try:
+      backend, op = get_backend_op_pair(t)
+    except LookupError:
+      # Linalg on tensors are WIP; explicitly ignore them for now.
+      if "linalg_on_tensors" in t:
```

Comment:
Seems like having tracking of what's covered by that path still wouldn't be bad? What about just adding this to the map?

Also, we really need proper target specifications that don't require all this custom mapping...

Reply:
+1 on target specification. maybe we could add labels that specify the new path vs old path in a way that's trivial to parse?

I think it would make sense to create two tables (something like `## XLA to IREE` and `## XLA to IREE via Linalg on Tensors`) store the coverage separately until we can delete the first table. The second table should only have LLVM and Vulkan as backend columns?

Reply:
I would actually avoid creating another column for this. It's not like that we will support these two paths in the future so this is not something similar to that we have LLVM dylib and Vulkan SPIR-V. I personally consider the op/model coverage as for external interested parties to keep track of the status in IREE. So internal refactoring meant to replace the existing Linalg-on-buffer path should not reveal there. :)

Reply:
Fair enough. I find this special casing a bit gross though :confused:

Reply:
Yeah. It's an intermediate state and I won't expect this to happen freqently. :)